### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ArtMot99**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.idea
+venv
+pyc
+db.sqlite3
```
Comment (ArsenPidhoretskyi) [inline .gitignore:4]: Add empty line

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.idea
+venv
+pyc
```
Comment (ArsenPidhoretskyi) [inline .gitignore:3]: This one must be 
```suggestion
*.pyc
```

Comment (ArsenPidhoretskyi) [reply .gitignore:3]: You does not have folder pyc)

File: taxi/models.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.db.models import UniqueConstraint
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255)
```
Comment (ArsenPidhoretskyi) [inline taxi/models.py:8]: You can just pass unique=True 
And not need constraints
The same with driver

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (ArtMot99) [reply .gitignore]: Without this restriction, I will not pass the tests)

**Review by EdAlekseiev — APPROVED** [review]
Good job

**Pull request closed without merging**